bandwidth-file-spec.txt

```diff
@@ -568,6 +569,41 @@
       The descriptor observed bandwidth mean for this relay in bytes per
       second.
 
+    "vote=" bool SP
+
+      [Zero or one time.]
+
+      If the value is 0, Tor directory authorities SHOULD ignore the relay's
+      entry in the bandwidth file. They SHOULD vote for the relay the same
+      way they would vote for a relay that is not present in the file.
+
+      This MAY be the case when this relay was not successfully measured but
+      the Bandwidth File, to diagnose why they were not measured.
```

Comment:
I think there are some words missing here.
"is included in the Bandwidth file"?

Reply:
yup, fixup